pkg/inst/tests/test_rdd.R

```diff
@@ -267,6 +267,20 @@ test_that("keyBy on RDDs", {
   expect_equal(actual, lapply(nums, function(x) { list(func(x), x) }))
 })
 
+test_that("repartition/coalesce on RDDs", {
+  rdd <- parallelize(sc, 1:10, 3L)
+  expect_equal(numPartitions(rdd), 3L)
```

Comment:
Could you also check how many elements are there in each partition using `collectPartition` ? Its okay to have it be a range (i.e. less than 5 and greater than 2 etc.) if its not deterministic.